ajax/libs/spa.js/package.json

```diff
@@ -0,0 +1 @@
+{"name":"spa.js","title":"SPA","version":"2.0.1","description":"A webapp framework for routing control and view transitions","directories":{"doc":"docs"},"repository":{"type":"git","url":"https://github.com/zhaoda/spa.git"},"keywords":["spa","webapp","javascript","framework"],"author":[{"name":"zhaoda","email":"zd6437@gmail.com","url":"http://zhaoda.net"},{"name":"linchao","email":"lin-chao@foxmail.com","url":"http://linchao.info"}],"license":"MIT","bugs":{"url":"https://github.com/zhaoda/spa/issues"},"homepage":"https://github.com/zhaoda/spa","devDependencies":{"grunt":"~0.4.2","grunt-contrib-clean":"~0.6.0","grunt-contrib-concat":"~0.5.0","grunt-contrib-copy":"~0.8.0","grunt-contrib-jshint":"~0.11.0","grunt-contrib-uglify":"~0.10.0","grunt-text-replace":"^0.4.0","gulp":"^3.9.0","gulp-clean":"^0.3.1","gulp-concat":"^2.6.0","gulp-header":"^1.7.1","gulp-jshint":"^1.11.2","gulp-rename":"^1.2.2","gulp-replace":"^0.5.4","gulp-uglify":"^1.4.1"},"scripts":{"test":"grunt","build":"gulp"},"npmName":"spa.js","npmFileMap":[{"basePath":"/dist","files":["**/*"]}],"filename":"spa-apis.js"}
```

Comment:
Please use [tools/fixFormat.js](https://github.com/cdnjs/cdnjs/blob/master/tools/fixFormat.js) to fix the format.